custom_message/custom_msgs/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>custom_msgs</name>
+  <version>1.0.0</version>
+  <description>
+    Custom Metric messages
+  </description>
+  <maintainer email="kaka@gmail.com">kaka</maintainer>
```

Comment:
This will be public, change to a reasonable email id. Same goes for author. Put opensource@rapyuta.io.